submissions/usides/js-dom/dist/js/main.js

```diff
@@ -0,0 +1,197 @@
+  },
+  {
+    number: 666,
+    nick: 'Katrin',
+    gender: 'Molly',
+    birth: '02.06.2014',
+    nationality: 'Russian',
+    marks: 'Sharp claws',
+    length: 50,
+    weight: 4,
+    color: 'Grey',
+    details:
+      '- Language: meowing, barking <br>- flip flops collector <br>- Charge with extreme caution',
+    charges:
+      '- Suspected in an assault with fangs and claws <br>- Terrorizes all dogs around ',
+    portret: 'crim_05.jpg',
+    files: ['crim_05_e01.jpg', 'crim_05_e02.jpg', 'crim_05_e03.jpg'],
+  },
+  {
+    number: 334,
+    nick: 'Sharpei',
+    gender: 'Tom',
+    birth: '01.03.2015',
+    nationality: 'American',
+    marks: 'Super cute',
+    length: 55,
+    weight: 4,
+    color: 'White / grey',
+    details:
+      '- Language: meowing <br>- calls himself as "sharp eye" but for some reason, other calls him as dog breed "shar-pei" <br>- master of disguise as turkey',
+    charges:
+      '- Suspected in mouses assassination with a rifle <br>- mouse killer for money - hitcat',
+    portret: 'crim_06.jpg',
+    files: ['crim_06_e01.jpg', 'crim_06_e02.jpg'],
+  },
+];
+
+const sideNavMenu = document.getElementById('side-nav-menu');
+const sideMenu = document.getElementById('side-nav');
+const menuBtn = document.getElementById('menu-btn');
+const nextBtn = document.getElementById('next-btn');
+let current = 0;
+
+document.addEventListener('DOMContentLoaded', function () {
+  loadMenu();
+  printData(current);
+});
+
+menuBtn.addEventListener('click', function (e) {
+  e.preventDefault();
+  sideMenu.classList.toggle('hidden');
+  this.classList.toggle('active');
+});
+
+nextBtn.addEventListener('click', function (e) {
+  e.preventDefault();
+  current++;
+  printData(current);
+  makeActive(current);
+  if (current == suspects.length - 1) current = -1;
+});
+
+sideNavMenu.addEventListener('click', function (e) {
+  e.preventDefault();
+  if (e.target.nodeName !== 'A') return;
+  window.scrollTo({
+    top: 0,
+    left: 0,
+    behavior: 'smooth',
+  });
+  let arrNum = suspects.findIndex(
+    (item) => item.number === Number(e.target.id.slice(1)),
+  );
+  printData(arrNum);
+  makeActive(arrNum);
+  current = arrNum;
+  sideMenu.classList.add('hidden');
+  menuBtn.classList.remove('active');
+});
+
+function loadMenu() {
+  const fragment = document.createDocumentFragment();
+  for (let i = 0; i < suspects.length; i++) {
+    const newElem = document.createElement('li');
+    newElem.innerHTML = `<a href="#" class="menu-link" id="p${
+      suspects[i].number
+    }">${labelCreate.call(suspects[i])}</a>`;
+    if (i == 0) newElem.firstElementChild.classList.add('active');
+    fragment.appendChild(newElem);
+  }
+  sideNavMenu.appendChild(fragment);
+  current = 0;
+}
+
+function printData(perp) {
+  let holders = document.querySelectorAll('[data-holder]');
+  for (let i = 0; i < holders.length; i++) {
+    holders[i].innerHTML = suspects[perp][holders[i].dataset.holder];
+  }
+  document
```

Comment:
I'd recommend placing these elements (`document.getElementById`) as constants at the beginning of this function. It would easier to read this function

Reply:
Done